roles/ansible-test-splitter/defaults/main.yaml

```
@@ -1,3 +1,3 @@
 ---
-ansible_test_splitter__test_changed: false
+ansible_test_splitter__test_changed: true
```

```suggestion
ansible_test_splitter__test_changed: false
```

We still want this to default to false for the full-test suite run during gating